modules/core/src/lighting/point-light.js

```diff
@@ -1,11 +1,7 @@
 import {PointLight as BasePointLight} from 'luma.gl';
-import {COORDINATE_SYSTEM} from '../lib';
 
 export default class PointLight extends BasePointLight {
```

Comment:
Is there other features planned for this class? Can just re-export luma's PointLight class?